### DISCUSSION TITLE
How to connect Nextcloud Snap 24.0.6 with own  Docker-DocumentServer

### DISCUSSION THREAD

**Discussion by vawaver** [ONLYOFFICE/Docker-DocumentServer#534] [Q&A]
I am not an -IT pro and I would like to ask for help.
This is the configuration of my Home lab server running on Proxmox.

VM01 - Ubuntu server 22.04 + NGINX Proxy server in docker
VM02 - Ubuntu server 22.04 + Nextcloud SNAP 24.0.6snap1 + OnlyOffice APP

There are 2 subdomains

-   nextcloud.mydomain.sk - Nextcloud SNAP (has its own SSL certificate but it goes through NGINX. No direct port 80,443 forwarding to VM02.)
-  onlyoffice.mydomain.sk - I have prepared this subdomain for Docker-DocumentServer and will be proxied via NGINX (if it is necessary.).

What is my goal:
1. I will be happy when I start Docker-DocumentServer on WM02 alongside the Nextclioud server. So I need the right docker command or docker-compose.yaml file.
2. If it is necessary - how to configure my NGINX Proxy manager properly?
3. OnlyOffice Docs settings (URL address) for the OnlyOffice app via subdomain (if it is necessary).

The Nextcloud server is reachable only via the web subdomain. No LAN users are allowed.

I tried something based on the info - https://helpcenter.onlyoffice.com/integration/nextcloud.aspx
But I did not succeed.

That is why I am looking for your support to help me with the right settings.
Thank you for your reply.

**Comment by ShockwaveNN** [comment]
@igwyd Please take a look

**Comment by avdddd1111** [comment]
Hello @vawaver sorry for the late reply. 
Regarding the issue with connecting Nextcloud Snap to Docker-Documents Server.

I've set up a similar configuration:

VM01: Ubuntu Server 22.04 + Nextcloud Snap, OnlyOffice Document Server in Docker.
VM02: Ubuntu Server 22.04 + Nginx Proxy Manager.


In Nginx Proxy Manager, I created hosts for redirection and enabled WebSocket support.

Everything is working for me.

If the issue is still relevant, could you describe any errors you encountered? Could you provide logs?

Additionally, you might want to check the following resources:

1. Docker commands: https://github.com/ONLYOFFICE/Docker-DocumentServer?tab=readme-ov-file#configuring-docker-image
2. Docker Compose for a separate document server: https://github.com/ONLYOFFICE/Docker-DocumentServer/blob/master/docker-compose.yml
3. Test deployment of Nextcloud + OnlyOffice: https://github.com/ONLYOFFICE/docker-onlyoffice-nextcloud
